index.js

```diff
@@ -1,45 +1,51 @@
 require('dotenv').config();
 
+async function getGuid(broadcasterLogin) {
+	console.log(`getGuid:${broadcasterLogin}`);
+	return fetch(`https://api.streamelements.com/kappa/v2/channels/${broadcasterLogin}`, {
+		headers: {
+			'Accept': 'application/json',
+			'Authorization': `Bearer ${jwtToken}`,
+			'Content-Type': 'application/json'
+		}
+	}).then(res => {
+		console.log(`channels/${login}: Response-Code ${res.status}`);
+		return res.json();
+	}).then(data => {
+		console.log(`channels/${login}: ${JSON.stringify(data)}`);
+		return data._id;
+	}).catch(err => {
+		console.log(`Error getting User Guid: ${err}`);
+	});
+}
+
 (async function () {
 if (process.env['JWT_TOKEN']) {
 	let jwtToken = process.env['JWT_TOKEN'];
-	let broadcasterLogin = process.env['BROADCASTER_LOGIN'] || 'me';
+	let guids = process.env['STREAMELEMENTS_ACCOUNT_IDS'] || 'me';
 	let textMessage = process.env['TEXT_MESSAGE'] || 'TEXT_MESSAGE not set!';
-	for (let login of broadcasterLogin.split(',')) {
-		console.log(`Login:${login}`);
-		await fetch(`https://api.streamelements.com/kappa/v2/channels/${login}`, {
+	for (let guid of guids.split(',')) {
+		if (guid == 'me') // TODO: Maybe check if it is a twitch username/login
```

Comment:
Maybe some logic like the one from https://stackoverflow.com/questions/67744689/is-there-some-trick-to-programmatically-determine-whether-a-given-twitch-usernam to check if it is a twitch login name might be helpful.
ChatGPT also suggested following logic, that I'd consider implementing (probably adjusting it a little bit):
```javascript
function identifyString(string) {
  // Regular expression patterns for Twitch login names and StreamElements account IDs
  const twitchPattern = /^[A-Za-z0-9_]{4,25}$/;
  const streamelementsPattern = /^[a-fA-F0-9]{24}$/;

  if (twitchPattern.test(string)) {
    return "Twitch login name";
  } else if (streamelementsPattern.test(string)) {
    return "StreamElements Account ID";
  } else {
    return "Unknown";
  }
}

// Example usage
const string1 = "mytwitchusername123";
const string2 = "5eb63bbbe01eeed093cb22bb8f5acdc3";
const string3 = "invalid_string";

console.log(identifyString(string1));  // Output: Twitch login name
console.log(identifyString(string2));  // Output: StreamElements Account ID
console.log(identifyString(string3));  // Output: Unknown
```

Here is a link to the source of the snippet above: https://chat.openai.com/share/a6c84ecd-b10f-42fc-8f26-1430fa8ad189